src/foapy/alphabet.py

```diff
@@ -0,0 +1,46 @@
+import numpy as np
+
+"""
+Implementation of ordered set - alphabet of elements.
+Alphabet is list of all unique elements in particular sequence.
+Parametres
+—--------
+X: array
+Array to get unique values.
+
+Returns
+—-----
+result: array.
+
+Examples
+—------
+
+"""
+
+
+def alphabet(X) -> np.ndarray:
+    result = []
+
+    for i in X:
+        if i not in result:
+            result.append(i)
+
+    return result
```

Comment:
Have you tried https://numpy.org/devdocs/reference/generated/numpy.unique.html#numpy-unique
function to get the alphabet?